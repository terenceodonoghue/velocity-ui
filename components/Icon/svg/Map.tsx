/** @jsx jsx */
import { jsx } from '@emotion/core';
import { IconProps } from '~/types';

const SvgMap: React.FunctionComponent<IconProps> = ({
  fill,
  theme,
  title,
  ...props
}) => (
  <svg viewBox="0 0 24 24" fill="none" {...props}>
    {title ? <title>{title}</title> : null}
    <path
      fillRule="evenodd"
      clipRule="evenodd"
      d="M21.4836 3.9089L15.2874 2.03087C15.2702 2.03087 15.253 2.03087 15.253 2.03087C15.2186 2.03087 15.2014 2.01364 15.167 2.01364C15.1325 2.01364 15.1153 2.01364 15.0809 2.01364C15.0465 2.01364 15.0293 2.01364 14.9948 2.01364C14.9604 2.01364 14.9432 2.01364 14.9088 2.03087C14.8916 2.03087 14.8744 2.03087 14.8744 2.03087L8.90189 3.85721L2.91222 2.03087C2.70568 1.96195 2.46472 2.01364 2.2926 2.13425C2.12048 2.27208 2 2.47884 2 2.70282V19.4156C2 19.7257 2.20654 20.0014 2.49914 20.0875L8.69535 21.9655C8.7642 21.9828 8.83305 22 8.90189 22C8.97074 22 9.03959 21.9828 9.10843 21.9655L15.0981 20.1564L21.0878 21.9655C21.1566 21.9828 21.2255 22 21.2943 22C21.4492 22 21.5869 21.9483 21.7074 21.8622C21.8795 21.7243 22 21.5176 22 21.2936V4.58085C22 4.27072 21.7935 4.01228 21.4836 3.9089ZM8.19621 20.3632L3.41136 18.9159V3.65045L8.19621 5.09774V20.3632ZM14.3924 18.8987L9.60757 20.3632V5.11497L14.3924 3.65045V18.8987ZM20.5886 20.3632L15.8038 18.9159V3.65045L20.5886 5.09774V20.3632Z"
      fill={fill || theme?.colors.lightBlueGrey}
    />
    <mask
      id="mask0"
      mask-type="alpha"
      maskUnits="userSpaceOnUse"
      x="2"
      y="2"
      width="20"
      height="20"
    >
      <path
        fillRule="evenodd"
        clipRule="evenodd"
        d="M21.4836 3.9089L15.2874 2.03087C15.2702 2.03087 15.253 2.03087 15.253 2.03087C15.2186 2.03087 15.2014 2.01364 15.167 2.01364C15.1325 2.01364 15.1153 2.01364 15.0809 2.01364C15.0465 2.01364 15.0293 2.01364 14.9948 2.01364C14.9604 2.01364 14.9432 2.01364 14.9088 2.03087C14.8916 2.03087 14.8744 2.03087 14.8744 2.03087L8.90189 3.85721L2.91222 2.03087C2.70568 1.96195 2.46472 2.01364 2.2926 2.13425C2.12048 2.27208 2 2.47884 2 2.70282V19.4156C2 19.7257 2.20654 20.0014 2.49914 20.0875L8.69535 21.9655C8.7642 21.9828 8.83305 22 8.90189 22C8.97074 22 9.03959 21.9828 9.10843 21.9655L15.0981 20.1564L21.0878 21.9655C21.1566 21.9828 21.2255 22 21.2943 22C21.4492 22 21.5869 21.9483 21.7074 21.8622C21.8795 21.7243 22 21.5176 22 21.2936V4.58085C22 4.27072 21.7935 4.01228 21.4836 3.9089ZM8.19621 20.3632L3.41136 18.9159V3.65045L8.19621 5.09774V20.3632ZM14.3924 18.8987L9.60757 20.3632V5.11497L14.3924 3.65045V18.8987ZM20.5886 20.3632L15.8038 18.9159V3.65045L20.5886 5.09774V20.3632Z"
        fill="white"
      />
    </mask>
    <g mask="url(#mask0)" />
  </svg>
);

export default SvgMap;
