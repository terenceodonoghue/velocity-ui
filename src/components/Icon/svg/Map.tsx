import React from 'react';
import { IconProps } from '../Icon';

const SvgMap: React.FunctionComponent<IconProps> = ({
  fill,
  size,
  theme,
  title,
  ...props
}) => (
  <svg viewBox={`0 0 ${size} ${size}`} fill="none" {...props}>
    {title ? <title>{title}</title> : null}
    <path
      fillRule="evenodd"
      clipRule="evenodd"
      d="M21.484 3.909L15.287 2.03h-.034c-.034 0-.052-.017-.086-.017h-.172c-.035 0-.052 0-.086.017h-.035L8.902 3.857l-5.99-1.826a.719.719 0 00-.62.103.744.744 0 00-.292.569v16.713c0 .31.207.585.5.671l6.195 1.878a.86.86 0 00.207.035.86.86 0 00.206-.035l5.99-1.809 5.99 1.81a.86.86 0 00.206.034.7.7 0 00.413-.138.744.744 0 00.293-.568V4.58c0-.31-.206-.569-.516-.672zM8.196 20.363l-4.785-1.447V3.65l4.785 1.448v15.265zm6.196-1.464l-4.784 1.464V5.115l4.784-1.465V18.9zm6.197 1.464l-4.785-1.447V3.65l4.785 1.448v15.265z"
      fill={fill || theme?.colors.lightBlueGrey}
    />
    <mask
      id="Map_svg__a"
      maskUnits="userSpaceOnUse"
      x={2}
      y={2}
      width={20}
      height={20}
    >
      <path
        fillRule="evenodd"
        clipRule="evenodd"
        d="M21.484 3.909L15.287 2.03h-.034c-.034 0-.052-.017-.086-.017h-.172c-.035 0-.052 0-.086.017h-.035L8.902 3.857l-5.99-1.826a.719.719 0 00-.62.103.744.744 0 00-.292.569v16.713c0 .31.207.585.5.671l6.195 1.878a.86.86 0 00.207.035.86.86 0 00.206-.035l5.99-1.809 5.99 1.81a.86.86 0 00.206.034.7.7 0 00.413-.138.744.744 0 00.293-.568V4.58c0-.31-.206-.569-.516-.672zM8.196 20.363l-4.785-1.447V3.65l4.785 1.448v15.265zm6.196-1.464l-4.784 1.464V5.115l4.784-1.465V18.9zm6.197 1.464l-4.785-1.447V3.65l4.785 1.448v15.265z"
        fill="#fff"
      />
    </mask>
  </svg>
);

export default SvgMap;
