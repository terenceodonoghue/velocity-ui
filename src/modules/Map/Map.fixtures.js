export const passenger = {
  avatarSrc: './users/nora.png',
  email: 'invision@invisionapp.com',
  interactions: 4,
  location: 'New York, NY',
  name: 'Nora M. Buchanan',
  phone: '+144–3412–4422',
};

export const trip = {
  distance: 12.3,
  energy: 42,
  from: { streetAddress: '37-27 74th Street', suburb: 'Jackson Heights' },
  price: 34.20,
  time: 12.4,
  to: { streetAddress: '81 Gate St Brooklyn', suburb: 'Greenpoint' },
};
