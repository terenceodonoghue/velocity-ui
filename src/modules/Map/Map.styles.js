import styled from '@emotion/styled';
import { Avatar, Container } from 'components';

export const Details = styled.div`
  flex: 0 0 253px;
`;

export const DetailsList = styled.dl`
  padding: 0 6px;
`;

export const FieldSet = styled.div`
  display: flex;
`;

export const Interactions = styled.p`
  margin: 0;
  color: var(--color-bluey-grey);
`;

export const Label = styled.dt`
  color: #b0bac9;
  margin-bottom: 10px;
  font-size: 12px;
  font-weight: var(--font-weight-medium);
  letter-spacing: 1.1px;
  text-transform: uppercase;
`;

export const Name = styled.p`
  margin: 0 0 2px;
  font-size: 15px;
  font-weight: var(--font-weight-medium);
  line-height: 1.47;
`;

export const Passenger = styled(Container.Card)`
  width: 655px;
`;

export const Payment = styled.div`
  flex: 1;
  display: flex;
  align-content: flex-start;
  flex-flow: row wrap;
  margin: -5px -12px -12px;
`;

export const PaymentLogo = styled.div`
  display: flex;
  align-items: center;
  justify-content: center;
  height: 75px;
  width: 140px;
  border-radius: 5px;
  border: solid 1px #e0e7ff;
  background-color: ${(props) => (props.active ? 'var(--color-clear-blue)' : null)};

  img {
    transform: scale(0.5);
  }
`;

export const PaymentType = styled.div`
  padding: 12px;
`;

export const Profile = styled.div`
  display: flex;
  margin: 9px 0 19px;
`;

export const ProfilePicture = styled(Avatar.Round)`
  height: 48px;
  width: 48px;
  transform: translate(6px, -2px);
`;

export const ProfileText = styled.div`
  margin-left: 15px;
`;

export const Value = styled.dd`
  margin: 0 0 20px;
  max-width: 241px;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
`;
