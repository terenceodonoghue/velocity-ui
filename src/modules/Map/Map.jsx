/** @jsx jsx */
import { jsx } from '@emotion/core';
import { number, shape, string } from 'prop-types';
import * as S from './Map.styles';

const Passenger = ({
  avatarSrc, email, interactions, location, name, phone,
}) => (
  <S.Passenger heading="Passenger info">
    <S.FieldSet>
      <S.Details>
        <S.Profile>
          <S.ProfilePicture src={avatarSrc}></S.ProfilePicture>
          <S.ProfileText>
            <S.Name>{name}</S.Name>
            <S.Interactions>{`${interactions} interaction${interactions === 1 ? '' : 's'}`}</S.Interactions>
          </S.ProfileText>
        </S.Profile>
        <S.DetailsList>
          <S.Label>Email</S.Label>
          <S.Value>{email}</S.Value>
          <S.Label>Phone</S.Label>
          <S.Value>{phone}</S.Value>
          <S.Label>Location</S.Label>
          <S.Value>{location}</S.Value>
        </S.DetailsList>
      </S.Details>
      <S.Payment>
        <S.PaymentType>
          <S.PaymentLogo><img alt="PayPal" src="./paypal.png" /></S.PaymentLogo>
        </S.PaymentType>
        <S.PaymentType>
          <S.PaymentLogo active><img alt="Visa" src="./visa.png" /></S.PaymentLogo>
        </S.PaymentType>
        <S.PaymentType>
          <S.PaymentLogo><img alt="Mastercard" src="./mastercard.png" /></S.PaymentLogo>
        </S.PaymentType>
        <S.PaymentType>
          <S.PaymentLogo><img alt="Apple Pay" src="./applepay.png" /></S.PaymentLogo>
        </S.PaymentType>
      </S.Payment>
    </S.FieldSet>
  </S.Passenger>
);

const Trip = ({
  distance, energy, from, price, time, to,
}) => (
  <S.Trip>
    {from.suburb}
    {from.streetAddress}
    {to.suburb}
    {to.streetAddress}
    <S.Stats>
      <S.Stat>
        <p>
          {distance}
          {' '}
          km
        </p>
        <p>Distance</p>
      </S.Stat>
      <S.Stat>
        <p>
          {time}
          {' '}
          min
        </p>
        <p>Time</p>
      </S.Stat>
      <S.Stat>
        <p>
          $
          {price}
        </p>
        <p>Price</p>
      </S.Stat>
      <S.Stat>
        <p>
          {energy}
          {' '}
          kWh
        </p>
        <p>Energy</p>
      </S.Stat>
    </S.Stats>
  </S.Trip>
);

Passenger.propTypes = {
  avatarSrc: string.isRequired,
  email: string.isRequired,
  interactions: number.isRequired,
  location: string.isRequired,
  name: string.isRequired,
  phone: string.isRequired,
};

Trip.propTypes = {
  distance: number.isRequired,
  energy: number.isRequired,
  from: shape({ streetAddress: string, suburb: string }).isRequired,
  price: number.isRequired,
  time: number.isRequired,
  to: shape({ streetAddress: string, suburb: string }).isRequired,
};

export default {
  Passenger,
  Trip,
};
