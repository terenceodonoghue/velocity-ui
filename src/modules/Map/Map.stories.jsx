import React from 'react';
import { storiesOf } from '@storybook/react';
import Map from './Map';
import { passenger, trip } from './Map.fixtures';

storiesOf('Modules|Map', module)
  .add('Passenger Info', () => (
    <Map.Passenger {...passenger} />
  ))
  .add('Trip Info', () => (
    <Map.Trip {...trip} />
  ));
