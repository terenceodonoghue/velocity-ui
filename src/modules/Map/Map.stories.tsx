import React, { FunctionComponent } from 'react';
import * as Map from './Map';
import * as fixtures from './Map.fixtures';

export default {
  title: 'Modules/Map',
};

export const PassengerInfo: FunctionComponent = () => (
  <Map.Passenger {...fixtures.passenger} />
);
export const TripInfo: FunctionComponent = () => (
  <Map.Trip {...fixtures.trip} />
);
