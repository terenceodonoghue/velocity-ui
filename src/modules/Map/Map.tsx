/** @jsx jsx */
import { jsx } from '@emotion/core';
import { FunctionComponent } from 'react';
import { Avatars, Selectable, Surfaces } from '../../components';
import * as css from './Map.styles';

type Address = {
  streetAddress: string;
  suburb: string;
};

interface PassengerProps {
  email: string;
  interactions: number;
  location: string;
  name: string;
  phone: string;
  src: string;
}

interface TripProps {
  distance: number;
  energy: number;
  from: Address;
  price: number;
  time: number;
  to: Address;
}

const PaymentLogo: FunctionComponent<Selectable> = ({ children, selected }) => (
  <div css={css.paymentLogo({ selected })}>{children}</div>
);

export const Passenger: FunctionComponent<PassengerProps> = ({
  email,
  interactions,
  location,
  name,
  phone,
  src,
}) => (
  <Surfaces.Card css={css.passenger} heading="Passenger info">
    <div css={css.passengerContent}>
      <div css={css.passengerDetails}>
        <div css={css.passengerProfile}>
          <Avatars.Round css={css.passengerAvatar} src={src} />
          <div css={css.passengerBio}>
            <p css={css.passengerName}>{name}</p>
            <p css={css.passengerInteractions}>
              {`${interactions} interaction${interactions === 1 ? '' : 's'}`}
            </p>
          </div>
        </div>
        <dl css={css.passengerFields}>
          <dt css={css.passengerFieldLabel}>Email</dt>
          <dd css={css.passengerFieldValue}>{email}</dd>
          <dt css={css.passengerFieldLabel}>Phone</dt>
          <dd css={css.passengerFieldValue}>{phone}</dd>
          <dt css={css.passengerFieldLabel}>Location</dt>
          <dd css={css.passengerFieldValue}>{location}</dd>
        </dl>
      </div>
      <div css={css.passengerPayment}>
        <div css={css.passengerPaymentType}>
          <PaymentLogo>
            <img alt="PayPal" src="./paypal.png" />
          </PaymentLogo>
        </div>
        <div css={css.passengerPaymentType}>
          <PaymentLogo selected>
            <img alt="Visa" src="./visa.png" />
          </PaymentLogo>
        </div>
        <div css={css.passengerPaymentType}>
          <PaymentLogo>
            <img alt="Mastercard" src="./mastercard.png" />
          </PaymentLogo>
        </div>
        <div css={css.passengerPaymentType}>
          <PaymentLogo>
            <img alt="Apple Pay" src="./applepay.png" />
          </PaymentLogo>
        </div>
      </div>
    </div>
  </Surfaces.Card>
);

export const Trip: FunctionComponent<TripProps> = ({
  distance,
  energy,
  from,
  price,
  time,
  to,
}) => (
  <Surfaces.Card css={css.trip}>
    <div css={css.tripIcons}>
      <div css={css.tripIcon}>
        <img alt="From" src="./check-blue.svg" />
      </div>
      <span css={css.tripIconDivider} />
      <div css={css.tripIcon}>
        <img alt="To" src="./marker-green.svg" />
      </div>
    </div>
    <div css={css.tripDetails}>
      <div css={css.tripAddress}>
        <p>{from.suburb}</p>
        <p>{from.streetAddress}</p>
      </div>
      <div css={css.tripAddress}>
        <p>{to.suburb}</p>
        <p>{to.streetAddress}</p>
      </div>
    </div>
    <div css={css.tripMetrics}>
      <div css={css.tripMetric}>
        <p>
          {distance}
          &nbsp;km
        </p>
        <p>Distance</p>
      </div>
      <div css={css.tripMetric}>
        <p>
          {time}
          &nbsp;min
        </p>
        <p>Time</p>
      </div>
      <div css={css.tripMetric}>
        <p>${price.toFixed(2)}</p>
        <p>Price</p>
      </div>
      <div css={css.tripMetric}>
        <p>
          {energy}
          &nbsp;kWh
        </p>
        <p>Energy</p>
      </div>
    </div>
  </Surfaces.Card>
);
