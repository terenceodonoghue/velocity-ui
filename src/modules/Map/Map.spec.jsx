/** @jsx jsx */
import { jsx } from '@emotion/core';
import { Passenger } from './Map';
import * as fixtures from './Map.fixtures';

describe('Map', () => {
  describe('Passenger', () => {
    it('matches snapshot', () => {
      const tree = renderer
        .create(withTheme(<Passenger {...fixtures.passenger} />))
        .toJSON();

      expect(tree).toMatchSnapshot();
    });

    it('uses correct plurality', () => {
      const component = renderer.create(
        withTheme(<Passenger {...fixtures.passenger} interactions={1} />),
      );

      expect(
        component.root.findAll(
          (el) =>
            el.type === 'p' &&
            el.children &&
            el.children[0] === '1 interaction',
        ),
      ).toHaveLength(1);

      component.update(
        withTheme(<Passenger {...fixtures.passenger} interactions={2} />),
      );

      expect(
        component.root.findAll(
          (el) =>
            el.type === 'p' &&
            el.children &&
            el.children[0] === '2 interactions',
        ),
      ).toHaveLength(1);
    });
  });
});
