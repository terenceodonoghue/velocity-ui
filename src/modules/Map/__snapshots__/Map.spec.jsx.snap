// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map Passenger matches snapshot 1`] = `
.emotion-24 {
  display: block;
  border-radius: 1px;
  border: solid 1px rgba(46,91,255,0.08);
  box-shadow: 0 10px 20px 0 rgba(46,91,255,0.07);
  padding: 32px 24px;
  background-color: #fff;
  width: 655px;
}

.emotion-0 {
  margin: 0 0 26px 0;
  color: #8798ad;
  font-size: 13px;
  font-weight: 400;
  -webkit-letter-spacing: 1.2px;
  -moz-letter-spacing: 1.2px;
  -ms-letter-spacing: 1.2px;
  letter-spacing: 1.2px;
  text-transform: uppercase;
}

.emotion-23 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-13 {
  -webkit-flex: 0 0 253px;
  -ms-flex: 0 0 253px;
  flex: 0 0 253px;
}

.emotion-5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: 9px 0 19px;
}

.emotion-1 {
  border-radius: 50%;
  height: 48px;
  width: 48px;
  -webkit-transform: translate(6px,-2px);
  -ms-transform: translate(6px,-2px);
  transform: translate(6px,-2px);
}

.emotion-4 {
  margin-left: 15px;
}

.emotion-2 {
  margin: 0 0 2px;
  font-size: 15px;
  font-weight: 500;
  line-height: 1.47;
}

.emotion-3 {
  margin: 0;
  color: #8798ad;
}

.emotion-12 {
  padding: 0 6px;
}

.emotion-6 {
  color: #b0bac9;
  margin-bottom: 10px;
  font-size: 12px;
  font-weight: 500;
  -webkit-letter-spacing: 1.1px;
  -moz-letter-spacing: 1.1px;
  -ms-letter-spacing: 1.1px;
  letter-spacing: 1.1px;
  text-transform: uppercase;
}

.emotion-7 {
  margin: 0 0 20px;
  max-width: 241px;
  white-space: nowrap;
  overflow: hidden;
  text-overflow: ellipsis;
}

.emotion-22 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-content: flex-start;
  -ms-flex-line-pack: start;
  align-content: flex-start;
  -webkit-flex-flow: row wrap;
  -ms-flex-flow: row wrap;
  flex-flow: row wrap;
  margin: -5px -12px -12px;
}

.emotion-15 {
  padding: 12px;
}

.emotion-14 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  height: 75px;
  width: 140px;
  border-radius: 5px;
  border: solid 1px #e0e7ff;
}

.emotion-14 img {
  -webkit-transform: scale(0.5);
  -ms-transform: scale(0.5);
  transform: scale(0.5);
}

.emotion-16 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  height: 75px;
  width: 140px;
  border-radius: 5px;
  border: solid 1px #e0e7ff;
  background-color: #2e5bff;
}

.emotion-16 img {
  -webkit-transform: scale(0.5);
  -ms-transform: scale(0.5);
  transform: scale(0.5);
}

<div
  className="emotion-24"
>
  <h3
    className="emotion-0"
  >
    Passenger info
  </h3>
  <div
    className="emotion-23"
  >
    <div
      className="emotion-13"
    >
      <div
        className="emotion-5"
      >
        <img
          className="emotion-1"
          src="https://s3.amazonaws.com/uifaces/faces/twitter/xravil/128.jpg"
        />
        <div
          className="emotion-4"
        >
          <p
            className="emotion-2"
          >
            Nora M. Buchanan
          </p>
          <p
            className="emotion-3"
          >
            4 interactions
          </p>
        </div>
      </div>
      <dl
        className="emotion-12"
      >
        <dt
          className="emotion-6"
        >
          Email
        </dt>
        <dd
          className="emotion-7"
        >
          invision@invisionapp.com
        </dd>
        <dt
          className="emotion-6"
        >
          Phone
        </dt>
        <dd
          className="emotion-7"
        >
          +144–3412–4422
        </dd>
        <dt
          className="emotion-6"
        >
          Location
        </dt>
        <dd
          className="emotion-7"
        >
          New York, NY
        </dd>
      </dl>
    </div>
    <div
      className="emotion-22"
    >
      <div
        className="emotion-15"
      >
        <div
          className="emotion-14"
        >
          <img
            alt="PayPal"
            src="./paypal.png"
          />
        </div>
      </div>
      <div
        className="emotion-15"
      >
        <div
          className="emotion-16"
        >
          <img
            alt="Visa"
            src="./visa.png"
          />
        </div>
      </div>
      <div
        className="emotion-15"
      >
        <div
          className="emotion-14"
        >
          <img
            alt="Mastercard"
            src="./mastercard.png"
          />
        </div>
      </div>
      <div
        className="emotion-15"
      >
        <div
          className="emotion-14"
        >
          <img
            alt="Apple Pay"
            src="./applepay.png"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
